.github/workflows/artifact-tests.yml

```diff
@@ -50,31 +50,37 @@ jobs:
       run: |
         echo "non-gzip-artifact-content=hello" >> $GITHUB_ENV
         echo "gzip-artifact-content=Some large amount of text that has a compression ratio that is greater than 100%. If greater than 100%, gzip is used to upload the file" >> $GITHUB_ENV
+        echo "empty-artifact-content=_EMPTY_" >> $GITHUB_ENV
 
     - name: Create files that will be uploaded
       run: |
-        mkdir artifact-path 
+        mkdir artifact-path
         echo ${{ env.non-gzip-artifact-content }} > artifact-path/world.txt
         echo ${{ env.gzip-artifact-content }} > artifact-path/gzip.txt
+        echo ${{ env.empty-artifact-content }} > artifact-path/empty.txt
```

Comment:
So the problem with this is that there will be a file created with `_EMPTY_` inside the content and the size won't be zero.

You can see for example here that artifact 3 is 8 bytes in size: https://github.com/actions/toolkit/actions/runs/1577186317

Something like `touch empty.txt` should create an empty file and that should be sufficient. It should have a size of 0. Here is an example: https://github.com/konradpabjan/artifact-test/actions/runs/1579111453

Afterwards you only need to check if the file exists, not necessarily the content.

Reply:
Oh, I forgot I refactored that to make it look like other cases; initially I did have it do a `touch`.

Reply:
Should be fixed now.